<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>214</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Pen price</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="le_pen_price"/>
     </item>
     <item>
      <widget class="QSpinBox" name="spinbox"/>
     </item>
     <item>
      <widget class="QLineEdit" name="le_total_pen_price"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Shugar price</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="le_shugar_price"/>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="dspinbox"/>
     </item>
     <item>
      <widget class="QLineEdit" name="le_total_shugar_price"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="lbl_result">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
